### PR TITLE
[Docs] Show opening tags.

### PR DESCRIPTION
**Pull request by sfadschm**
Now standalone.

[x] Show opening tags for php examples and closing tags for mixed html.

### CODE REVIEW

**Comment by sfadschm** [comment]
This will be reset after the other PRs, please do not review yet 😄

File: user_guide_src/source/installation/upgrade_security/002.php [hunk 1]
```diff
@@ -6,6 +6,7 @@
 );
 
 // ...
```
Comment (kenjis) [inline user_guide_src/source/installation/upgrade_security/002.php:8]: It seems `// ...` is not needed.
```suggestion
```
Screenshot 2022-02-25 19 07 35

File: user_guide_src/source/libraries/pagination/004.php [hunk 2]
```diff
@@ -16,5 +16,7 @@ public function index()
 }
 
 // In the views:
+?>
+
 <?= $pager->links('group1') ?>
 <?= $pager->simpleLinks('group2') ?>
```
[the remark this answers is not in the document]
Comment (sfadschm) [reply user_guide_src/source/libraries/pagination/004.php:22]: Should we just remove the comment?
Or maybe make it a HTML comment so it sticks with the HTML?

[the remark this answers is not in the document]
Comment (kenjis) [reply user_guide_src/source/libraries/pagination/004.php:22]: Making a HTML comment is better?

**Comment by sfadschm** [comment]
Sorry for the mess, cannot squash on mobile 😂
Maybe squash when merging.

**Review by kenjis — APPROVED** [review]

**Comment by sfadschm** [comment]
Thanks @kenjis for the thorough reviews. The next PR for syntax fixes will take some days.

**Comment by kenjis** [comment]
@sfadschm Okay, have a nice weekend!

**Pull request merged**